rllib/policy/torch_policy_v2.py

```diff
@@ -203,6 +208,13 @@ def __init__(
         self.batch_divisibility_req = self.get_batch_divisibility_req()
         self.max_seq_len = max_seq_len
 
+        # if model is an RLModule it won't have tower_stats instead there will be a
```

Comment:
nit: capitalize "If", add "." at end of comment.

Reply:
done.